### ISSUE TITLE
Too hard to follow up the documentation

### ISSUE THREAD

**Issue by bob4gd** [MicrosoftDocs/sql-docs#6027]
The documentation is fragmented in sections, each one in a different html page, that you have to click and wait to access.  This mechanism is very bad to read them at the internet due to a "long"delay, that distract the reader from what he is reading.

Thus, the documentation exists and is there, but to effectively read it's really hard.

I recommend to compose more sections in the same page so the delay does not affect the user experience and he can effectively read the documentation faster.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9e84e5bc-943c-a96a-642d-90d7dac07e53
* Version Independent ID: 27d085ff-2fcf-4057-08f4-ad3f36675a56
* Content: [Creating an OLE DB Driver for SQL Server Application - SQL Server](https://docs.microsoft.com/en-us/sql/connect/oledb/ole-db-driver/creating-a-oledb-driver-for-sql-server-application?view=sql-server-ver15)
* Content Source: [docs/connect/oledb/ole-db-driver/creating-a-oledb-driver-for-sql-server-application.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/connect/oledb/ole-db-driver/creating-a-oledb-driver-for-sql-server-application.md)
* Product: **sql**
* Technology: **connectivity**
* GitHub Login: @David-Engel
* Microsoft Alias: **v-daenge**

**Comment by WilliamAntonRohm** [comment]
@bob4gd -- thank you for your considered feedback.  You are welcome to submit issues related to overall site experience, or suggestions for improved site functionality or design, at [https://aka.ms/sitefeedback/](https://aka.ms/sitefeedback).
